test/custom_test_data/member_row.yml

```diff
@@ -0,0 +1,9 @@
+:url: 'http://www.parliament.go.tz/mps-list'
```

Comment:
I've raised an issue for this: https://github.com/everypolitician/scraper_test/issues/13